src/itdelatrisu/opsu/beatmap/OszUnpacker.java

```diff
@@ -83,8 +83,7 @@ public boolean accept(File dir, String name) {
 		files = null;
 
 		if (!dirs.isEmpty()) {
-			String text = String.format("Imported %d new beatmap pack%s.", dirs.size(), dirs.size() == 1 ? "" : "s");
-			UI.getNotificationManager().sendNotification(text);
+			UI.getNotificationManager().sendNotification("ui.notifications.beatmaps.imported", dirs.size());
```

Comment:
Added the "hack" for plurals since the formatting implementation allow us to do so.

> By default, `String.format()` ignores the extra arguments if the string does not contain enough formatting tags.

> **But**, if `String.format()` has more formatting tags than the arguments, it will throw a `MissingFormatArgumentException`

> This also applies if the formatting tag is of the incorrect type, but throws a different exception. (e.g. using `%d` instead of `%s` on a `String` format argument)